org/allPRs.ts

```diff
@@ -16,8 +16,29 @@ export const internalLink = async () => {
     }
 }
 
+export const xcodeprojConfiguration = async () => {
+    if (danger.github.thisPR.repo == "macos-browser") {
```

Comment:
If, in the future, we'll have more rules that apply only to some repos, we may want to move the check to the default function to make it easier to figure out what rules are called for a certain repo, but, for now, I think this works better.

Reply:
Oh yes, that’s a great idea for later 👍